lib/round.rb

```diff
@@ -0,0 +1,53 @@
+class Round
+  attr_reader :deck, :turns, :correct_answers
+  def initialize(deck)
+     @deck = deck
+     @turns = []
+     @correct_answers = 0
+  end
+
+  def current_card 
+    deck.cards.shift
```

Comment:
I might consider doing the actual shift in the `take_turn` method so that the deck is only changing after a turn is done. Right now, anytime the `current_card` method is called, it will shift the deck. The way the program is set up, this seems to work fine, but if you ever called current_card more than once between turns, I think you'd end up skipping cards. That way current card method is only responsible for returning the top card on the deck and nothing more.